core/templates/dev/head/components/social_buttons/SocialButtonsDirective.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the Social Buttons.
+ */
+
+oppia.directive('socialButtons', [function() {
+  return {
+    templateUrl: 'components/socialButtons',
```

Comment:
There should be 'element' and 'scope' specified in the keys. Why was it removed? Both are important -- the first is needed to ensure this is only used for elements, and the second is needed to ensure that this directive is isolated from its parent scopes.